<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_EnginePreferences</class>
 <widget class="QWidget" name="GUI_EnginePreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="rb_auto">
     <property name="text">
      <string>Automatic</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rb_pulse">
     <property name="text">
      <string notr="true">Pulseaudio</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rb_alsa">
     <property name="text">
      <string notr="true">Alsa</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
